step-generation/src/commandCreators/compound/distribute.ts

```diff
@@ -108,6 +110,22 @@ export const distribute: CommandCreator<DistributeArgs> = (
   )
   const { pipette } = args
 
+  const isWasteChute =
+    invariantContext.additionalEquipmentEntities[args.dropTipLocation] !=
+      null &&
```

Comment:
```suggestion
    invariantContext.additionalEquipmentEntities[args.dropTipLocation] !=
      null &&
```
I could be missing something here, but do we need these nullish checks for isWasteChute and isTrashBin?

Reply:
good call, i think this is redundant. Thanks!